test/Microsoft.NET.Publish.Tests/GivenThatWeWantToPublishAHelloWorldProject.cs

```diff
@@ -175,6 +175,74 @@ public static void Main()
                 .And
                 .HaveStdOutContaining("Hello from a netcoreapp2.0.!");
         }
+        
+        [Theory]
+        [InlineData("win-arm")]
+        [InlineData("win8-arm")]
+        [InlineData("win81-arm")]
+        [InlineData("win10-arm")]
+        public void Publish_standalone_post_netcoreapp2_arm_app(string runtimeIdentifier)
+        {
+            // Tests for existence of expected files when publishing an ARM project
+            // See https://github.com/dotnet/sdk/issues/1239
+
+            if (UsingFullFrameworkMSBuild)
+            {
+                //  Disabled on full framework MSBuild until CI machines have VS with bundled .NET Core / .NET Standard versions
+                //  See https://github.com/dotnet/sdk/issues/1077
+                return;
+            }
+
+            var targetFramework = "netcoreapp2.0";
+
+            TestProject testProject = new TestProject()
+            {
+                Name = "Hello",
+                IsSdkProject = true,
+                TargetFrameworks = targetFramework,
+                RuntimeIdentifier = runtimeIdentifier,
+                IsExe = true,
+            };
+            
+
+            testProject.SourceFiles["Program.cs"] = @"
+using System;
+public static class Program
+{
+    public static void Main()
+    {
+        Console.WriteLine(""Hello from an arm netcoreapp2.0.!"");
+    }
+}
+";
+            var testProjectInstance = _testAssetsManager.CreateTestProject(testProject);
+
+            testProjectInstance.Restore(Log, testProject.Name);
+            var publishCommand = new PublishCommand(Log, Path.Combine(testProjectInstance.TestRoot, testProject.Name));
+            var publishResult = publishCommand.Execute();
+
+            publishResult.Should().Pass();
+
+            var publishDirectory = publishCommand.GetOutputDirectory(
+                targetFramework: targetFramework,
+                runtimeIdentifier: runtimeIdentifier);
+            var selfContainedExecutable = $"Hello{Constants.ExeSuffix}";
```

Comment:
`{Constants.ExeSuffix}` - this won't work correctly when this test is run on OSX or Linux.  When running on those platforms, `Constants.ExeSuffix` will be string.Empty.  However, since the project always targets Windows, the output self-contained executable will always be for Windows.  Thus no matter which platform this test is run on, the output assembly will be named `Hello.exe`.

So I think you can just hard-code `.exe` here.